### PR TITLE
Layout Component and Divider Theme Updates

### PR DESCRIPTION
**Pull request by rtkaaho**
- Adding simple layout component
- Modifying the divider theme to be more flexible
- Adding Classname to typography components

Screen Shot 2020-02-24 at 12 02 37 PM

### CODE REVIEW

File: src/components/layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    height: 100%;
+  `}
+`;
+
+export const Layout: LayoutComponent<LayoutProps> = ({
+  children,
+  className,
+}) => {
+  return (
+    <React.Fragment>
```
Comment (rtktpeng) [inline src/components/layout/Layout.tsx]: Why do we need a `<React.Fragment>` wrapping here?

Comment (rtkaaho) [reply src/components/layout/Layout.tsx]: good catch. Needed it earlier during implementation. Dont need it anymore!

Comment (rtkmrussell) [inline src/components/layout/Layout.tsx]: Does this fragment do anything?  Can't you just return the Container?

Comment (rtkaaho) [reply src/components/layout/Layout.tsx]: Removed that fragment in that earlier commit there after tom commented :)

**Comment by rtkaaho** [comment]
> Looks reasonable to me 👍 Do we want to start using CSS grid over flexbox?

I think for containers like this flex box is great. We will see if we need to change it in the future as more use cases start coming in

File: src/components/layout/Layout.tsx (hunk 1 above)
Comment (rtkmrussell) [reply src/components/layout/Layout.tsx]: Sorry... I'm too slow 😊

**Pull request merged**